src/features/tree-base/js/tree-base.js

```diff
@@ -715,7 +715,7 @@
         };
 
         rowHeaderColumnDef.visible = grid.options.treeRowHeaderAlwaysVisible;
-        grid.addRowHeaderColumn( rowHeaderColumnDef, -100 );
+        grid.addRowHeaderColumn(rowHeaderColumnDef, -100, true);
```

Comment:
I added a new attribute to prevent build columns from being triggered multiple times in a row. ColumnDefs is now responsible for building the columns. See line 763 of [Grid.js](https://github.com/mportuga/ui-grid/blob/75f98c3eda3e46f14f9d924e6f5a6d6e35bcc2b5/src/js/core/factories/Grid.js#L763)